client/app/bundles/course/assessment/question/programming/components/OnlineEditorCppView.intl.js

```diff
@@ -0,0 +1,137 @@
+import { defineMessages } from 'react-intl';
```

Comment:
If the entire file is the same, would it be better we have a common set of translations for all languages, then add in additional language specific messages when necessary?